source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
```

Comment:
[style guide] We typically refer to tab names with the `:guilabel:` directive!
```suggestion
You can rename diagrams on the :guilabel:`Mapping` tab of Relational Migrator.
```

Reply:
Done!